City-of-Helsinki/outdoors-sports-map
# Feature/hulk 26 add outdoor services

## Description
Show outdoor services:
 - provides outdoor service choice as `Hiking` which contains options to choose
 - adds own icon for Hiking and options
 - shows a service on the map when selected, together with chosen sport
 - shows details of a outdoor service when clicked on a map
 - options include `Cooking facility`, `Information point`, `Outdoor / ski lodge`, `Lapp tent shelter / earth lodge` and `
Camping`

## Context

[HULK-26](https://project.anders.fi/browse/HULK-26)

## How Has This Been Tested?
 - check outdoor service is provided in the UI
 - when selecting `Hiking`, options are provided 
 - selecting an option should show a place on the map with own icon along with chosen sport
 - selecting a place from the map should show details information with icon

## Manual Testing Instructions for Reviewers
  - check supporting service is provided in the UI
 - when selecting `Hiking`, options are provided 
 - selecting an option should show a place on the map with own icon along with chosen sport
 - selecting a place from the map should show details information with icon
 
## Screenshots

Outdoor service on UI:

![outdoor_services_in_ui](https://user-images.githubusercontent.com/117433931/228751806-4c8ffb1b-58f1-4f68-a7ce-dfcb2cb821e1.png)

shows selected outdoor service along with chosen sport:

![map_showing_outdoor_service_and_selected_sport_together](https://user-images.githubusercontent.com/117433931/228751963-59b6ada3-c862-4b53-ad04-e9c39dd31f01.png)

details of an outdoor service:

![detail_of_selected_outdoor_service](https://user-images.githubusercontent.com/117433931/228752046-c61cc97c-96b9-4473-a6b6-50563888ffcb.png)
